sys/action.js

```diff
@@ -296,11 +399,21 @@ module.exports = (options) => {
                         operationId: 'mwApiQuery'
                     }
                 },
+                '/rawquery': {
+                    all: {
+                        operationId: 'mwRawApiQuery'
+                    }
+                },
                 '/siteinfo': {
                     all: {
                         operationId: 'mwApiSiteInfo'
                     }
                 },
+                '/sitematrix': {
```

Comment:
I wanna reiterate a question that was lost somewhere along the review process - what's the advantage of using the sitematrix instead of already implemented siteinfo?

Reply:
@Pchelolo I answered that [here](https://github.com/wikimedia/restbase/pull/887#discussion_r144390020), don't have any arguments beyond that. I'l replace it with siteinfo.